includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1807,6 +1807,7 @@ public function get_level3_data_from_order( WC_Order $order ): array {
 
 			$description     = substr( $item->get_name(), 0, 26 );
 			$quantity        = $item->get_quantity();
+			$quantity        = is_int( $quantity ) ? $quantity : 1; // In case of float type value of quantity, the value should be 1.
```

Comment:
Why not just round of instead of setting it to one?

Reply:
Casting to `integer` wouldn't work, because if the quantity is less than `1`, e.g. `0.4`, it will result in `0` which is wrong. I have updated the code and added an additional unit test for that.